packages/cli/src/commands/generate/service/templates/test.ts.template

```diff
@@ -36,24 +47,24 @@ describe('${pluralCamelName}', () => {
     expect(result).toEqual(scenario.${singularCamelName}.one)
   })
 
-  <% if (create) { %>scenario('creates a ${singularCamelName}', async (${stringifyButNotScenario(create).includes('scenario.') ? 'scenario: StandardScenario' : ''}) => {
+  <% if (create) { %>scenario('creates a ${singularCamelName}', async (${stringifyValue(create).includes('scenario.') ? 'scenario: StandardScenario' : ''}) => {
     const result = await create${singularPascalName}({
-      input: ${stringifyButNotScenario(create)},
+      input: ${stringifyValue(create)},
     })
 
     <% for (const [name, value] of Object.entries(create)) { %>
-    expect(result.${name}).toEqual(${stringifyButNotScenario(value)})<% } %>
+    expect(result.${name}).toEqual(${stringifyValue(value, true)})<% } %>
```

Comment:
When reading this line it's not obvious what `true` does.

And now that I think about it, anything with "stringify" in the function name is a bit misleading, isn't it? Because it's not actually stringifying all of it anymore 😆 Unless we call it `stringifyButNotScenarioAndSometimesNotDate` 😁 Just kidding!

But I do think we could maybe come up with a better function name. And also don't be afraid to have more methods if that makes sense. Or an options argument, so you see the name of the argument you're setting, like `newFunctionName(value, { stringifyDates: false })`

Reply:
this `newFunctionName(value, { stringifyDates: false })` does not seem to be an option within the templates. i overloaded the function. please let me know what you think